thirdparty/chart-tp-api-svc/participant-list.svc.yaml

```
@@ -0,0 +1,213 @@
+##
+# The following is values.yaml file for a ttk that
+# mocks out the participant-list-svc features for
+# the 3P API
+#
+# The actual config for the /services/{Type} API comes from an external repo: 
+# https://github.com/vessels-tech/ml-oss-sandbox This is a workaround because 
+# we can't specify inline TTK configs from within this values file. 
+##
+
+COMPUTED VALUES:
```

```suggestion
COMPUTED VALUES: {}
```

or should this be `[]`?

OR is there an indent issue here?